model/task/db.go

```diff
@@ -463,6 +463,13 @@ func scheduleableTasksQuery() bson.M {
 	}
```

Comment:
Can you add a test of the scheduler?

Reply:
I believe this is entirely covered by existing tests.